# PACMod system watchdog report2

std_msgs/Header header

bool accel_rpt_timeout
bool brake_rpt_timeout
bool brake_decel_rpt_timeout
bool cabin_climate_rpt_timeout
bool cabin_fan_speed_rpt_timeout
bool cabin_temp_rpt_timeout
bool cruise_control_rpt_timeout
bool dash_left_rpt_timeout
bool dash_right_rpt_timeout
bool engine_brake_rpt_timeout
bool hazard_lights_rpt_timeout
bool headlight_rpt_timeout
bool horn_rpt_timeout
bool marker_lamp_rpt_timeout
bool media_controls_rpt_timeout
bool parking_brake_rpt_timeout
bool rear_pass_door_rpt_timeout
bool shift_rpt_timeout
bool sprayer_rpt_timeout
bool steering_rpt_timeout
bool turn_rpt_timeout
bool wiper_rpt_timeout
bool pacmod1_sanity_fault
bool pacmod2_sanity_fault
bool pacmod3_sanity_fault
bool pacmini1_sanity_fault
bool pacmini2_sanity_fault
bool pacmini3_sanity_fault
bool pacmod1_component_rpt_timeout
bool pacmod2_component_rpt_timeout
bool pacmod3_component_rpt_timeout
bool pacmini1_component_rpt_timeout
bool pacmini2_component_rpt_timeout
bool pacmini3_component_rpt_timeout
bool pacmod1_system_present_fault
bool pacmod2_system_present_fault
bool pacmod3_system_present_fault
bool pacmini1_system_present_fault
bool pacmini2_system_present_fault
bool pacmini3_system_present_fault
bool drive_mode_invalid
bool global_cmd_sanity_fault
bool global_cmd_timeout
bool exhaust_brake_rpt_timeout